scripts/rconcli.sh

```diff
@@ -13,8 +13,21 @@ run_rcon_cli() {
         ew ">>> RCON is not enabled. Aborting RCON command ..."
         exit
     fi
-    output=$(rcon -c "$RCON_CONFIG_FILE" "${cmd}")
-    ei_nn "> RCON: "; e "${output}"
+
+    if [[ ! -f "${RCON_CONFIG_FILE}" ]]; then
+        er ">>> RCON config file not found. Aborting RCON command ..."
+        return
+    fi
+
+    # Edge case for broadcast because it doesn't support spaces in the message
+    if [[ ${cmd,,} == broadcast* ]]; then
```

Comment:
Perhaps we can use get the first command at the top of the function using `$1`, then use `shift` and `$*` to collect the rest of the parameters `$cmd` without using variable expansion?

Reply:
It's a nice approach too, no doubt.
It would mean a refactor of the function, so I think you should have it in a PR just about refactoring `rconcli` with the timestamps and this change.